src/main/java/com/github/krzysiekjodlowski/ox/Move.java

```diff
@@ -0,0 +1,45 @@
+package com.github.krzysiekjodlowski.ox;
+
+import com.github.krzysiekjodlowski.ox.model.FieldNumber;
+import com.github.krzysiekjodlowski.ox.model.Symbol;
+
+/**
+ * Player move representation as Event.
+ *
+ * @author Krzysztof Jodlowski
+ */
+public class Move implements Event<Move> {
+  private final FieldNumber fieldNumber;
+  private final Symbol playersSymbol;
+
+  public Move(FieldNumber fieldNumber, Symbol playersSymbol) {
+    this.fieldNumber = fieldNumber;
+    this.playersSymbol = playersSymbol;
+  }
+
+  public FieldNumber getFieldNumber() {
+    return this.fieldNumber;
+  }
+
+  public Symbol getPlayersSymbol() {
+    return this.playersSymbol;
+  }
+
+  @Override
+  public Move getData() {
+    return this;
+  }
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
Due to errorProne:
- Classes that override equals should also override hashCode.

Reply:
Thats right, done